extensions/shared/external-media/media-browser/media-item.js

```diff
@@ -16,6 +16,15 @@ function MediaItem( props ) {
 		}
 	}, [ props.onClick ] );
 
+	// Catch space and enter keypresses
+	const onKeyDown = e => {
+		if ( e.which === 13 || e.which === 32 ) {
+			// Prevent spacebar from scrolling the page down
+			e.preventDefault();
+			onClick( event );
```

Comment:
Let's stay consistent with naming event variables `event`. It looks like in this case it's also needed to not pass an undefined variable to `onClick()`?

Reply:
Doh. Fixed.